test/Microsoft.Identity.Abstractions.Tests/CredentialDescriptionTest.cs

```diff
@@ -461,5 +454,50 @@ public void CopyConstructorFromNull_ShouldThrow()
             _ = Assert.Throws<ArgumentNullException>(() => new CredentialDescription(original!));
 #pragma warning restore CS8600 // Converting null literal or possible null value to non-nullable type.
         }
+
+        [Fact]
+        public void Algorithm_SetAndGet_ShouldWork()
+        {
+            /*
+            // <algorithm_json>
+            {
+                "ClientCredentials": [
+                {
+                    "Algorithm": "RS256",
```

Comment:
Still not right. This only works with a certificate, @RojaEnnam
you need to take one of the cases with a certificate (for instance KeyVault)

Reply:
Addressed. Thanks.